suite-native/module-accounts-import/src/accountsImportSlice.ts

```diff
@@ -0,0 +1,39 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+import { networks } from '@suite-common/wallet-config';
+
+import { AssetItem } from './components/SelectableAssetList';
+
+export const actionPrefix = '@accountsImport';
+
+export type AccountsImportState = {
+    selectedCurrency: AssetItem;
```

Comment:
What about `selectedNetwork`? Currency envokes fiat imo

```suggestion
    selectedNetwork: AssetItem;
```

Reply:
nit: You're using coin and currency interchangebly and it's little bit confusing

Reply:
I changed naming to `network` everywhere where it makes sense (variables, component name...) and unified it. e21e5713ac754d22544aedeb5ae901093cf708a7